planet/api/orders.py

```diff
@@ -123,30 +182,44 @@ async def get_order(self, order_id):
         order = Order(resp.json())
         return order
 
-    async def cancel_order(self, order_id):
+    async def cancel_order(
+        self,
+        order_id: str
+    ) -> Response:
         '''Cancel a queued order.
 
-        According to the API docs, cancel order should return the cancelled
-        order details. But testing reveals that an empty response is returned
-        upon success.
+        **Note:** According to the API docs, cancel order should return the
+        cancelled order details. But testing reveals that an empty response is
```

Comment:
Have you already alerted the orders API team of this issue?

Reply:
I believe so... @sarasafavi have we alerted them yet?

Reply:
taking this offline as something to check into